kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverterTest.java

```diff
@@ -174,7 +176,28 @@ public void testInteger() {
     kafkaConnectStruct.put(fieldName, fieldDoubleValue);
     kafkaConnectRecord = spoofSinkRecord(kafkaConnectSchema, kafkaConnectStruct);
 
-    bigQueryTestRecord = new BigQueryRecordConverter().convertRecord(kafkaConnectRecord);
+    bigQueryTestRecord = new BigQueryRecordConverter().convertRecord(kafkaConnectRecord, SHOULD_CONVERT_DOUBLE);
+    assertEquals(bigQueryExpectedRecord, bigQueryTestRecord);
+  }
+
+  @Test public void testDoubleSpecial() {
```

Comment:
Should verify -Infinity and NaN as well. Can probably be done in a loop in this test.